CONTRIBUTING.md

```diff
@@ -1,13 +1,40 @@
 
-Thank you for considering a contribution to Subiquity.  Here's what we like to
-see:
-
-* Contributors will need to have signed the
-  [CLA](https://ubuntu.com/legal/contributors/agreement).
-* Format, lint, unit, integration, and API tests should be passing.
-  * format - either run `make format` or `pre-commit install`.
-    See [pre-commit](https://pre-commit.com/#install) for more details on that
-    tool.
+Thank you for considering a contribution to Subiquity.  Here's some things to
+note:
+
+## Code of Conduct
+
+This project is subject to the [Ubuntu Code of Conduct](https://ubuntu.com/community/code-of-conduct)
+to foster an open and welcoming place to contribute. By participating in the
+project (in the form of code contributions, issues, comments, and other
+activities), you agree to abide by its terms.
```

Comment:
One thing I thought about adding was a recommendation or at least a mention for users to go sign the code of conduct with a link to https://help.ubuntu.com/community/SigningCodeofConduct

Although as it stands I don't think it's a requirement to submit bugs and the language states implicit agreement by a users participation. So it's strictly not necessary, but it felt like a semi-useful addition.  What are your thoughts?